pymc_marketing/model_config.py

```diff
@@ -369,10 +383,66 @@ def handle_parameter_distributions(
     }
 
 
+def create_hierarchical_non_center(
+    name: str,
+    distribution_kwargs: dict[str, Any],
+    **kwargs,
+) -> pt.TensorVariable:
+    """
+    Create a hierarchical non-centered distribution.
+
+    This function constructs a hierarchical non-centered distribution using the provided
+    distribution parameters for offset, mu, and sigma. It returns a deterministic variable
+    representing the hierarchical non-centered distribution.
+
+    Parameters
+    ----------
+    name : str
+        The name of the variable.
+    distribution_kwargs : dict[str, Any]
+        A dictionary containing the distribution parameters for 'offset', 'mu', and 'sigma'.
+    **kwargs
+        Additional keyword arguments, including 'dims' for specifying desired dimensions.
+
+    Returns
+    -------
+    pt.TensorVariable
+        A PyMC deterministic variable representing the hierarchical non-centered distribution.
+
+    """
+    desired_dims = kwargs.get("dims", ())
+
+    mu_dist = distribution_kwargs["mu"]
+    mu_dims = mu_dist.get("dims", ())
+    sigma_dist = distribution_kwargs["sigma"]
+    sigma_dims = sigma_dist.get("dims", ())
+
+    offset = pm.Normal(name="offset", mu=0, sigma=2, dims=desired_dims)
+
+    mu_global = create_distribution(
+        f"{name}_mu_global",
```

Comment:
{name}_mu is how centered is so use that instead

Reply:
solve!